### ISSUE TITLE
New Swarm not communicating with Docker Cloud

### ISSUE THREAD

**Issue by zbruhnke** [docker/for-aws#64]
### Expected behavior
When I was using an older version of Docker For AWS(17.03) the external load balancer would have ports open for 2376, 2375, etc
### Actual behavior
When I setup a new stack today(17.06) it only has port 7 opened by default on the ELB
### Information
Is this intended functionality? When I'm trying to connect to docker cloud (using Bring your own swarm) it seems to be losing communication rather quickly and I'm not sure if it has to do with onlt having port 7 open or what is going on.
### Steps to reproduce the behavior

Use the Docker for AWS template for 17.06 CE

I used 3 managers and 2 workers all m4.large in size

**Comment by kencochrane** [comment]
Yes, that is the expected behavior, you do not want to have ports 2375 and 2376 exposed over the ELB that would be a security vulnerability. The ELB is for exposing ports for your services (website, etc).

When you say you are losing communication, can you give some more details, so we can help figure out what is going on.

/cc @amegianeg @bernardopericacho

**Comment by zbruhnke** [comment]
I can communicate fine if I ssh into a manager but if I run the docker cloud proxy and try to communicate through docker for mac I get this error
Cannot connect to the Docker daemon at unix:///blah/some_folders. Is the docker daemon running?

meaning I can get to the swarm and it shows as being connected but any command I try to run with docker I get that error

**Comment by FrenchBen** [comment]
@zbruhnke When you go into the Docker Cloud dashboard, under swarm, do you see that your swarm is connected?
There should be a service running on your cluster that is cloud specific, can you share the logs of that service?
